webapp/src/properties/types.tsx

```diff
@@ -75,3 +71,22 @@ export abstract class PropertyType {
     abstract type: PropertyTypeEnum
     abstract displayName: (intl: IntlShape) => string
 }
+
+export abstract class DatePropertyType extends PropertyType {
+    isDate = true
+    canGroup = false
```

Comment:
There is a lot of extra fields that are set by default for the DatePropertyType, that could be inherited from PropertyType. Everything except isData, getDateFrom and getDateTo, is the same in PropertyType (name and value).